content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Goodbye Slack: GNU Radio is moving into the Matrix"
+author: "Martin Braun"
+date: "2020-04-27"
+sponsored: "0"
+aliases: ["blog/moving-to-the-matrix"]
+---
+
+With the growth of a project there come changes. For GNU Radio, a big change is
+in the pipeline too: We're moving from our chat service from Slack to
+[Matrix](https://matrix.gnuradio.org).
+Untypically for us, this decision came about pretty quickly, but without much
+community involvement. Let me go through why did this, how we reached a decision,
+and, most importantly, how you can join the new chat service as well.
+
+<!--more-->
+
+
+## Stop talking, and let me chat: Instructions for the impatient
+
+If you're already done reading, all you need to do is head over to
+[matrix.gnuradio.org](https://matrix.gnuradio.org) and create an account. Once
```

Comment:
... unless you already have an account on a Matrix homeserver with federation; then you can just join us on [#gnuradio:gnuradio.org](https://matrix.to/#/!PiiKkGTcBDLmPnxhoT:gnuradio.org?via=gnuradio.org&via=atrz.de&via=matrix.org).